src/webrtc/groupCallEventHandler.ts

```diff
@@ -61,13 +65,16 @@ export class GroupCallEventHandler {
             dataChannelOptions = { ordered, maxPacketLifeTime, maxRetransmits, protocol };
         }
 
-        return new GroupCall(
+        const groupCall = new GroupCall(
             this.client,
             room,
             callType,
             content?.dataChannelsEnabled,
             dataChannelOptions,
         );
+        groupCall.groupCallId = content["conf_id"];
```

Comment:
This can use the state key instead of `conf_id` as outlined here https://github.com/matrix-org/matrix-doc/pull/3401/files#r716981077